### ISSUE TITLE
Markup tag breaks output renderer

### ISSUE THREAD

**Issue by Bilge** [benweet/stackedit#1076]
```
Heading 1
---------

<div>

Heading 2
---------

</div>

Heading 3
---------
```

Everything before and everything inside the `div` does not render in the output pane. Heading 3 renders normally. On GitHub all three render normally.

**Comment by benweet** [comment]
Closing this. Please open a new ticket if the problem still exists in the v5 (https://stackedit.io/app).

**Comment by Bilge** [comment]
@benweet Thanks for working on a fix in the new version. LGTM 👍 

However, in my particular case I am using the `align="center"` attribute on the `div` tag, which does not cause the text inside the `div` to become centered even though it does on GitHub. Do you want to support the *align* attribute or not?

Aside, I also noticed large images that do not fit the editor's viewport are not scaled down in the new version any more (they are rendered 1:1). Do you want me to create an issue to track this or are you still working on it?

**Comment by benweet** [comment]
This works fine to me in the v5:

```
<div align="center">
Centered
</div>
```

Ok for the scale down of the images. It will be fixed in the next release.

**Comment by Bilge** [comment]
Yessir, that does indeed work. Here's where it doesn't, though:

```md
<div align="center">

## Heading

Body

</div>
```

I suspect it may be a consequence of mixing markup and markdown.

**Comment by benweet** [comment]
Yes absolutely, though you can have what you want with plain HTML:

```
<h2 align="center">Heading</h2>

<div align="center">
Body
</div>
```

**Comment by Bilge** [comment]
Again, and I hate to imply it's some kind of gold standard (because it's not), but it is working on GitHub as I wrote it. Since I am ostensibly writing in markdown I prefer to use the minimum amount of markup necessary.

I am indifferent as to whether you choose to support this curious mix of the two syntaxes. In v4 it did not render at all (or it caused the rest of the document to become corrupted) which was a critical issue. In v5 the only problem is it's not centred, which is not really a big problem for me, and I understand if you have more important things to do. Nevertheless it would be nice if it did work the same way as on GitHub ;)

**Comment by benweet** [comment]
Ok that's a bug..
